src/app/components/App.jsx

```diff
@@ -33,6 +33,64 @@ const pageRequiresEntropy = path => {
     return entropyPages.indexOf(page) !== -1;
 };
 
+class WelcomePanel extends React.Component {
```

Comment:
this should live in its own file

Reply:
I agree in principle, as it's a private component (not used elsewhere, not exported) I'm ok with it here until there's more bandwidth.